app/editor/src/features/admin/evening-overview/EveningOverview.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { FaBinoculars, FaPaperPlane } from 'react-icons/fa';
+import { MdAdd } from 'react-icons/md';
+import { useEveningOverviews } from 'store/hooks/admin/useEveningOverviews';
+import { Button, ButtonVariant, Row } from 'tno-core';
+
+import { IEveningOverviewSection } from './interfaces';
+import { OverviewSection } from './overview-section';
+import * as styled from './styled';
+
+/** Evening overview section, contains table of items, and list of overview sections */
+export const EveningOverview: React.FC = () => {
+  const handleAdd = () => {
+    setSections((prev) => [
+      ...prev,
+      {
+        name: 'New section',
+        items: [],
+      },
+    ]);
+  };
+  const [sections, setSections] = React.useState<IEveningOverviewSection[]>([]);
+  const [, api] = useEveningOverviews();
+  React.useEffect(() => {
```

Comment:
We have this issue on most of our pages.  If the API goes down or it returns `0` items then this will have an infinite loop.

Reply:
Debugging this right now..